ckan/logic/action/create.py

```diff
@@ -610,7 +610,7 @@ def member_create(context, data_dict=None):
                               table_id=obj.id,
                               group_id=group.id,
                               state='active')
-
+        member.group = group
```

Comment:
No, with the 1.1 upgrade not explicitly assigning the group to the new member object led to the relation between the two not being set at the database level, which caused loads of [test failures](https://circleci.com/gh/ckan/ckan/2679?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) due to Authorization errors